instance/models/openedx_instance.py

```diff
@@ -111,10 +115,7 @@ def domain(self):
 
         Returns external domain if present, otherwise falls back to internal domain.
         """
-        if self.external_lms_domain:
-            return self.external_lms_domain
-        else:
-            return self.internal_lms_domain
+        return self.get_domain('lms')
```

Comment:
@e-kolpakov As far as I can tell, the strings only occurs once in the code, so moving this single occurrence to a different place does not seem to help.  Moreover, we already have protection from typos, since line 134 will fail if we use a string that does not have a corresponding attribute name.